content/en/data_security/real_user_monitoring.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Real User Monitoring Data Security
+kind: documentation
+aliases:
+    - /real_user_monitoring/security/
+further_reading:
+- link: "/data_security/"
+  tag: "Documentation"
+  text: "Review the main categories of data submitted to Datadog"
+- link: "/data_security/synthetics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Data Security"
+- link: "/real_user_monitoring/session_replay/privacy_options/"
+  tag: "Documentation"
+  text: "Session Replay Privacy Options"
+- link: "https://www.datadoghq.com/blog/default-privacy-session-replay/"
+  tag: "Blog"
+  text: "Obfuscate user data with Session Replay default privacy settings"
+---
+
+<div class="alert alert-info">This page is about the security of data sent to Datadog. If you're looking for cloud and application security products and features, see the <a href="/security/" target="_blank">Security</a> section.</div>
+
+## Overview
+Real User Monitoring (RUM) provides controls for implementing privacy requirements and ensuring organizations of any scale do not expose sensitive or personal information. Data is stored on Datadog-managed cloud instances and encrypted at rest. The default behaviors and configurable options described on this page are designed to protect end user privacy and prevent sensitive organizational information from being collected. Learn more about [Privacy at Datadog][13].
+## Compliance frameworks
+RUM can be configured for compliance with many standards and regulatory frameworks, including, but not limited to:
+
+- GDPR
+- HIPAA 
+- ISO
+- CCPA/CPRA
+
+## Privacy restrictions
+By default, there are some privacy restrictions in place that protect user data to help comply with regulatory and standards frameworks.
+
+### Browser RUM use of cookies
+Browser RUM requires **first party cookies** to be enabled on an end user's browser to collect data. If required by the jurisdictions in which you operate, your pages should be configured so that the end user must accept cookies RUM is initialized and begins collecting data.
```

Comment:
> your pages should be configured so that the end user must accept cookies RUM is initialized and begins collecting data.

Should that be "must accept cookies **before** RUM is initialized"?